resources/deployments/mongo/device_deployments_test.go

```diff
@@ -286,7 +378,6 @@ func TestAggregateDeviceDeploymentByStatus(t *testing.T) {
 		session := db.Session()
 		store := NewDeviceDeploymentsStorage(session)
 
-		// deployments are created with status DeviceDeploymentStatusPending
```

Comment:
This was not true. In this test, each DeviceDeployment were specifically created with different statuses to cover all of them